EventsExpress-SoftServe/EventsExpress
# Implement leaflet map to choose location

dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress/issues/126)


## Code reviewers

- [x] @iuriikhrystiuk 

### Second Level Review

- [x] @sand0 

## Summary of issue

The opportunity to use Google maps within the event's creation so I can simplify the choice of location

## Summary of change

- Created table for location coordinates storing (latitude, longitude)
- Implemented leaflet map
- Implemented reverse geocoding coordinates
- Created unit-tests for location service

## Testing approach

- Check event creation
- Check edit event

## CHECK LIST
- [x]  Сode coverage >=34.1%
